src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt

```diff
@@ -0,0 +1,23 @@
+package vegait.rs.osipodgorica.configuration
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+@Configuration
+internal class ResourceWebConfiguration : WebMvcConfigurer {
+
+    @Value("\${app.file.resource-location:file:uploads/}")
```

Comment:
Kakva je ovo sintaksa sa ovim  slashom \?

Reply:
Kotlin

Reply:
https://stackoverflow.com/questions/46238790/how-to-use-spring-annotations-like-autowired-or-value-in-kotlin-for-primitive